client/src/modules/list/components/Items/ListItemName/ListItemName.jsx

```diff
@@ -67,11 +74,12 @@ class ListItemName extends PureComponent {
     } = this.props;
     const isNameUpdated = updatedName !== name;
     const canBeUpdated = updatedName.trim().length > 1;
+    const formattedName = formatName(userName, formatMessage);
```

Comment:
do we have to call `formatName` on the current User name? current user can not be anonymous...